obspy/imaging/waveform.py

```diff
@@ -548,6 +548,9 @@ def _plot_event(self, event):
         else:
             time = event["time"]
             text = event["text"] if "text" in event else None
+            marker = event["marker"] if "marker" in event else "*"
+            marker_color = event["marker_color"] in event else "yellow"
+            marker_size = event["marker_size"] in event else 12
```

Comment:
If we make modifications here..
 - a) we need to stick to matplotlib kwarg naming conventions of the `Axes` plotting method that is used (e.g. `markersize` in `Axes.plot()` -- not `marker_size`)
 - b) we might want to right away allow any modifications in the used matplotlib plotting method, i.e. pass through arbitrary kwargs (i.e. pass through anything left in `event` after extracting `time` and `text` to matplotlib)
 - c) if there's a chance that eventually we need to pass through some other kwargs to some other underlying routine, we should group the matplotlib kwargs (like `markersize`) into a subdictionary in `event`, e.g. `event['plot_kwargs`] = {'markersize': 15}` (e.g. other people might want to modify the `annotate()` call or the plot call for picks..)
 - d) maybe we should check if it makes sense to switch the plotting routine from `plot()` to `scatter()` (but maybe that's not really necessary, just thought that scatter might be in principle more appropriate)

a) is a must from my point of view
b) is also strongly recommended I think
c) dunno.. maybe not that important but it's a decision we need to consider right now
d) dunno.. maybe not important

Reply:
'a' makes sense to me. I think it also makes sense to implement 'c'. I definitely would like to be able to customize the marker, but I think it might be useful to people in the future to customize other things like the text. 'd' It probably does make sense in principal but is less of a priority to me personally.

Reply:
@megies I'm new to crowd-sourced coding an making edits to larger projects on Github. I've only used Github as version control for my own code. Nevertheless, I can take a crack at implementing changes a & c. Is that a good path forward?

Reply:
Sure, go ahead. Might be too big of a change to make it in online editor though.. maybe you should switch to working on your local computer in whatever IDE you normally use.